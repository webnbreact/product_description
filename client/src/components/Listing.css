.button {
  color: #199094;
  font-weight: bold;
}

.button:hover {
  cursor: pointer;
  text-decoration: underline;
}

.containerDiv {
  color:#585a3a;
  font-family: 'Montserrat', sans-serif;
  margin: 0 17%;
  max-width: 510px;
}

.home_location {
  margin: 0px;
  word-wrap: break-word;
  font-size: 16px;
  font-weight: 400;
  line-height: 1.375em;
  color: #484848;
}

.home_name {
  font-size: 32px;
  font-weight: 800;
  line-height: 1.125em;
  color: #484848;
  padding-top: 6px;
  padding-bottom: 6px;
}

.home_type {
  box-sizing: border-box;
  font-weight: 800;
  font-size: 12px;
  font-weight: bold;
  line-height: 1.3333333333333333em;
  text-transform: uppercase;
} 
