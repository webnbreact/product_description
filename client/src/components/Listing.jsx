import React from 'react';
import axios from "axios";

class Listing extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      listingId: 1,
      listingInfo: {}

    }
  }
  componentDidMount() {
    axios
      .get('http://localhost:1128/listing', {
        params: {
          listingId: this.state.listingId
        }
      })
      .then(data => {
        console.log('this is data', data.data);
        this.setState({
          listingInfo: data.data
        })
      })
      .catch(error => {
        console.log('axios get failed', err);
      });
  }
  render()
}

export default Listing;