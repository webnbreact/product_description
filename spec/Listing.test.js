import React from 'react';
import { shallow } from 'enzyme';

import Listing from '../client/src/components/Listing.jsx';

describe('Listing', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Listing debug/>)
    expect(component).toMatchSnapshot();
  });
});