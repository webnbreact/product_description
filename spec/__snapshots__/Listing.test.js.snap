// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Listing Test should render 1`] = `<div />`;
